pymodbus/framer/base.py

```diff
@@ -18,15 +19,16 @@ class ModbusFramer:
 
     name = ""
 
-    def __init__(self, decoder, client=None):
+    def __init__(self, decoder, client=None) -> None:
```

Comment:
Would be nice to type decoder and client as well.

Reply:
decoder I could do in f7b7517bc9dc5a3df5b01220ecb95b331fa579c6, client creates a circular import :(